src/components/ProductCard/ProductCard.jsx

```diff
@@ -1,21 +1,30 @@
 import React, { useState } from "react";
+import { Link, useNavigate } from "react-router-dom";
+import { useCart } from "../../context/cart-context";
 
 import "./ProductCard.css";
 export function ProductCard(props) {
-  const [isHovered, setIsHovered] = useState(false);
-  const { title, image, price, categoryName, stars, badge } = props;
+  const navigate = useNavigate();
+  const { title, image, price, categoryName, stars, badge, id } = props;
+  const { addToCart, cartData } = useCart();
+  const Token = localStorage.getItem("Manazone.Token");
```

Comment:
Usually the first letter is capital only for components, for regular variables use ```token``` instead.

Reply:
Thanks for pointing, I have changed it.